libs/openFrameworks/graphics/ofCairoRenderer.cpp

```diff
@@ -19,7 +19,10 @@ _cairo_status ofCairoRenderer::stream_function(void *closure,const unsigned char
 }
 
 ofCairoRenderer::ofCairoRenderer()
-:graphics3d(this){
+:graphics3d(this)
```

Comment:
move graphics3d() initializer at the end of initailizers list to avoid reorder warning